CIP-0036/README.md

```diff
@@ -110,7 +110,7 @@ considered valid if the following conditions hold:
   The advised way to construct a nonce is to use the current slot number.
   This is a simple way to keep the nonce increasing without having to access
   the previous transaction data.
-- The reward address is a Shelley address discriminated for the same network
+- The reward address is a Shelley payment address discriminated for the same network
```

Comment:
That sentence reads funny now 🙃 ... "reward_address is a payment address". Naming things is hard I guess. 

-> Might be good to consider renaming the field altogether.

Reply:
maybe we should change it here and in the cddl from `rewards_address` to `voting_rewards_payout_address`

Reply:
I'm on the fence about renaming it, I would like further input.

Historically, before Catalyst's Fund 4, this field was a payment address and was named `rewards_address`, see #89 - it does not appear to have been an issue back then. With that said, I do understand additional the potential for confusion as this is not the first time this field has changed.

Reply:
@KtorZ @gitmachtl 

See most recent commit, I have renamed the field to just `payment_address`. In addition to making things clearer, I believe removing the notion of rewards from the field name makes the specification less Catalyst specific; as the notion of voting rewards are Catalyst specific.